lib/manual_testing.py

```diff
@@ -0,0 +1,30 @@
+import copy
+import typing
+import edn_format
+import json
+from . import edn_syntax
+
+
+def get_root_node_from_path(path: str):
+    try:
+        # Data is wrapped with "" and has escaped all the "s, this de-escapes
+        data_with_wrapping_string_removed = json.load(
+            open(path, 'r'))
+        root_data_immutable = edn_format.loads(
+            data_with_wrapping_string_removed)
+        root_data = typing.cast(
+            dict, edn_syntax.convert_edn_to_pythonic(root_data_immutable))
+        root_node = root_data[":symphony"]
+    except:
+        root_data_immutable = edn_format.loads(open(path, 'r').read())
+        root_node = typing.cast(
+            dict, edn_syntax.convert_edn_to_pythonic(root_data_immutable))
```

Comment:
Only reason this is complicated is because the inputs folder files are inconsistent. I'd prefer they all be pure EDN (no wrapping quotes) so my IDE can help me read them, but I'm lazy / don't want to send in a bigger diff in this PR

Reply:
TODO: clean up the inputs folder